examples/basic/annotations/slope.py

```diff
@@ -10,21 +10,24 @@
 
 from bokeh.models import Slope
 from bokeh.plotting import figure, show
+from bokeh.palettes import Sunset10
 
 # linear equation parameters
 slope, intercept = 2, 10
 
 xpts = np.arange(0, 20, 0.2)
 ypts = slope * xpts + intercept + np.random.normal(0, 4, 100)
 
-p = figure(width=450, height=450, x_axis_label='x', y_axis_label='y',
+blueshade, yellowshade = Sunset10[0], Sunset10[5]
```

Comment:
Same as the other PR, let's use names `blue` and `yellow` here
```suggestion
blue, yellow = Sunset10[0], Sunset10[5]
```